R/tostring.R

```diff
@@ -36,29 +36,54 @@ default_hsep <- d_hsep_factory()
 
 
 #' @rdname tostring
+#'
 #' @inheritParams MatrixPrintForm
-#' @param widths (proposed) widths for the columns of \code{x}
+#' @param widths numeric (or NULL). (proposed) widths for the columns of
+#'  \code{x}. The expected length of this numeric vector can be retrieved with
+#'  `ncol() + 1` as the column of row names must also be considered.
 #' @param hsep character(1). Characters to repeat to create header/body
-#' separator line.
-#' @exportMethod toString
+#'  separator line.
+#' @param tf_wrap logical(1). Should the texts like title and footnotes be wrapped?
+#' @param max_width integer(1) (or NULL). If NULL, it will be assigned to
```

Comment:
This description doesn't mention that it only applies to title and footer materials and not the body of the table.